bin/kmd

```diff
@@ -68,20 +67,16 @@ def main(args):
     if args.dry_run:
         return
 
-    print('Installing {} to {}'.format(args.release, args.prefix))
-    install_root = os.path.join(args.prefix, args.release, 'root')
-
-    komodo.shell('{1} {0} .{0} {0}'.format(args.release, args.renamer))
```

Comment:
if `renamer` is no longer in use (perhaps it is, I don't know) then it should be deleted.

Reply:
It is not in use - I have removed it.